jsapp/js/account/plan.component.tsx

```diff
@@ -100,8 +107,28 @@ export default function Plan() {
     [state.products, state.organization, state.subscribedProduct]
   );
 
-  const hasManageableStatus = useCallback((subscription: BaseSubscription) =>
-    activeSubscriptionStatuses.includes(subscription.status), []);
+  const freeTierOverride = useMemo((): FreeTierOverride | null => {
+    if (envStore.isReady) {
+      const thresholds = envStore.data.free_tier_thresholds as FreeTierThresholds;
+      const display = envStore.data.free_tier_display as FreeTierDisplay;
+      const featureList : {[key: string]: string | null} = {};
+      display.feature_list.forEach((feature, key) => {
+        featureList[`feature_list_${key+1}`] = feature;
+      });
+      return {
+        name: display.name,
+        ...thresholds,
+        ...featureList,
+      };
+    }
+    return null;
+  }, [envStore.isReady]);
+
+  const hasManageableStatus = useCallback(
```

Comment:
This is a useCallback because you want it to be a function that accepts various subscriptions right? Just checking my own understanding, I don't think there is a change to make.

Reply:
Yep!